cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/Operand.qll

```diff
@@ -46,7 +46,7 @@ class Operand extends TStageOperand {
   /**
    * Gets the location of the source code for this operand.
    */
-  final Language::Location getLocation() { result = this.getAnyDef().getLocation() }
+  final Language::Location getLocation() { result = this.getUse().getLocation() }
```

Comment:
I think you just undid your change 🙂

Reply:
whoops! The sync script got me :laughing: